src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -7,7 +7,10 @@
     /// <summary>
     /// <see cref="ILoggerProvider"/> implementation that creates returns instances of <see cref="ApplicationInsightsLogger"/>
     /// </summary>
+#if !NETSTANDARD2_0
+    // For NETSTANDARD2.0 We take dependency on Microsoft.Extensions.Logging.ApplicationInsights which has ApplicationInsightsProvider having the same ProviderAlias and don't want to clash with this ProviderAlias.
     [ProviderAlias("ApplicationInsights")]
+#endif
     internal class ApplicationInsightsLoggerProvider : ILoggerProvider
```

Comment:
Why do you keep this type instead of completely switching to Microsoft.Extensions.Logging.ApplicationInsights implementation?

Reply:
We cannot remove this whole class - as that'd be a breaking change for AI customers, and we don't want that. (except when we change major from to 3.XX).